core/build.gradle.kts

```diff
@@ -47,6 +48,13 @@ kotlin {
             }
         }
 
+        val iosArm32Main by getting {
+            dependencies {
+                api(coroutines("core", version = "1.5.0-native-mt!!"))
```

Comment:
Per #129, Kable no longer depends on `-native-mt` Coroutines and expects library consumers to properly pull in the `-native-mt` Coroutines for targets that need it.

```suggestion
```